### DISCUSSION TITLE
bindsym crashes i3wm

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#4926] [Q&A]
Hello

while trying to play with config I found a bug or have a misconception of wm work.

```
bindsym --release $mod+t exec --no-startup-id lxterminal --command='bash -ic "echo Some command"'
```
When binding executes, i3wm restarts and I see login screen.

Can you please, suggest, why?

```
$ i3 -version
i3 version 4.19.1-non-git © 2009 Michael Stapelberg and contributors
```

**Comment by Airblader** [comment]
Please open an issue for bug reports, it'll ask you for a bunch more information. We'll need a log file at least, more likely a backtrace to see whether i3 is really crashing, and if so, why.

**Reply by ghost** [reply]
done

**Reply by ghost** [reply]
> Please open an issue for bug reports, it'll ask you for a bunch more information. We'll need a log file at least, more likely a backtrace to see whether i3 is really crashing, and if so, why.

https://github.com/i3/i3/issues/4927

Do you think that newer version will solve the issue?

**Reply by Airblader** [reply]
There was no crash bug fix in 4.20 that sticks out, so it's at least not super likely that it'd be fixed in 4.20. I'm OK if you want to stay on 4.19, but without a log file and back trace there's not much we can do unless someone can reproduce it.
